setup.py

```diff
@@ -42,7 +42,8 @@
         'keyring == 19.0.1',
         'invoke == 1.2.0',
         'pyyaml == 5.1',
-        'requests == 2.21.0'
+        'requests == 2.21.0',
+        'psutil == 5.7.0'
```

Comment:
This file isn't used anymore. Suggest installing psutil next to keyring

Reply:
which file isn't used anymore? requests?

Reply:
I was using it to make sure the script installed everything it needed to add the key to the keyring.
I changed it to explicitly pip install the packages we really need (psutil and keyring)